Reply/app/src/main/java/com/example/reply/ui/ReplyListContent.kt

```diff
@@ -0,0 +1,221 @@
+package com.example.reply.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.FavoriteBorder
+import androidx.compose.material.icons.filled.StarBorder
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import com.example.reply.R
+import com.example.reply.data.Email
+
+@Composable
+fun ReplyListOnlyContent(
+    replyHomeUIState: ReplyHomeUIState,
+    modifier: Modifier = Modifier
+) {
+    LazyColumn(modifier = modifier) {
+        items(replyHomeUIState.emails) { email ->
+            ReplyEmailListItem(email = email)
+        }
+    }
+}
+
+@Composable
+fun ReplyListAndDetailContent(
+    replyHomeUIState: ReplyHomeUIState,
+    modifier: Modifier = Modifier,
+    selectedItemIndex: Int
+) {
+    Row(modifier = modifier, horizontalArrangement = Arrangement.spacedBy(12.dp)) {
+        LazyColumn(modifier = modifier.weight(1f)) {
+            items(replyHomeUIState.emails) { email ->
+                ReplyEmailListItem(email = email)
+            }
+        }
+        LazyColumn(modifier = modifier.weight(1f)) {
+            items(replyHomeUIState.emails[selectedItemIndex].threads) { email ->
+                ReplyEmailThreadItem(email = email)
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ReplyEmailListItem(
+    email: Email,
+    modifier: Modifier = Modifier.padding(horizontal = 16.dp, vertical = 4.dp)) {
+    Card(modifier = modifier,) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(20.dp)
+        ) {
+            Row(modifier = Modifier.fillMaxWidth()) {
+                ReplyProfileImage(
+                    drawableResource = email.sender.avatar,
+                    description = email.sender.fullName,
+                )
+                Column(
+                    modifier = Modifier
+                        .weight(1f)
+                        .padding(horizontal = 12.dp, vertical = 4.dp),
+                    verticalArrangement = Arrangement.Center
+                ) {
+                    Text(
+                        text = email.sender.firstName,
+                        style = MaterialTheme.typography.labelMedium
+                    )
+                    Text(
+                        text = "20 mins ago",
```

Comment:
Is it fine to hardcode this?

Reply:
Yes I let me update data class to accommodate it.